### DISCUSSION TITLE
Fans control macro makes printer screen response weird

### DISCUSSION THREAD

**Discussion by robomopa** [Guilouz/Creality-Helper-Script-Wiki#645] [Issues]
When fans control macro is installed, the cooling fan sliders not work as they should be. Model fan respond slow, side and aux fan sliders jumps to a slower speed when adjusted by sliding. For example you set %40 but it jumps back to 20 and working at that speed. When the fans control macro uninstalled, screen works ok.

**Comment by Andy3dprinter** [comment]
Yup. I had the same issue. Seems like there is an issue with main printer.cfg and with the fans. I ended up going into my old CFG and copying the fan stuff and then pasting it into the newest one after update and it fixed the issue. I did not look into what had changed but it did fix it

**Reply by robomopa** [reply]
uninstalling the macro from helper script solves it.
